src/website/src/app/documentation/demos/toggles/ng/basic.html

```diff
@@ -1 +1 @@
-<input type="checkbox" clrToggle />
```

Comment:
idk whats going on here or why this is even here. This isn't part of this PR and It will be reverted.